dothething/dothething/ClipsView.swift

```diff
@@ -35,71 +34,101 @@ struct ClipsView: View {
     @Binding var toggle: Bool
     
     var body: some View {
-        VStack {
-            HStack {
-                // back button
-                Button(action: {
-                    print("Back button tapped")
-                    clipsViewModel.backButtonPressed()
-                    toggle.toggle()
-                }) {
-                    Text("Back")
-                }                                                                                                                                                                                                                                                                                                                                                                                                                                   
-                .padding(.leading)
-                .frame(maxWidth: .infinity, alignment: .leading)
+        GeometryReader { geometry in
+            VStack {
+                ZStack {
+                    HStack {
+                        Button(action: {
+                            print("Back button tapped")
+                            clipsViewModel.backButtonPressed()
+                            toggle.toggle()
+                        }) {
+                            Image(systemName: "chevron.left")
+                                .font(.system(size: 24))
+                                .foregroundColor(.accentColor)
+                        }
+                        .padding(.leading)
+                        Spacer()
+                    }
+                    
+                    Text("domino")
+                        .font(.custom("Montserrat-Medium", size: 27))
+                        .foregroundColor(Color.accentColor)
+                        .tracking(8)
+                        .multilineTextAlignment(.center)
+                }
```

Comment:
The changes in this section introduce a new UI layout, including a back button, a centered title, and a button for either uploading or sharing the rally. This reorganization of the UI elements improves the overall structure and visual appeal of the ClipsView. The use of GeometryReader ensures that the layout adapts to different screen sizes.